src/forms/EditDocumentForm.js

```diff
@@ -18,72 +17,53 @@ const EditDocumentForm = ({
+  Object.keys(versionState).forEach(function(e) {
+    options.push({
+      key: e,
+      value: e,
+      text: versionState[e].title,
+      content: (
+        <button
+          onClick={() => onVersionStatusChange(e)}
+          type="button"
+          tabIndex="0"
+          key={e}
         >
-          Document Title:
-        </label>
+          <Badge state={e} />
+        </button>
+      ),
+    });
+  });
+  return (
+    <Form>
+      <Form.Field required>
+        <label>Document Title:</label>
         <input
           data-testid="name-input"
-          className="Form--Input border border-grey row-1 cell-9"
           type="text"
           name="file_name"
           placeholder="Phenotypic Data manifest for..."
           defaultValue={fileName}
           onChange={e => onNameChange(e)}
         />
-        <label
-          className="Form--Label row-2 cell-3 text-right"
-          htmlFor="file_name"
-        >
-          Approval Status:
-        </label>
-        <div className="row-2 cell-9">
-          <div className={dropdownClass}>
-            <button
-              type="button"
-              className="Dropdown--button justify-start pl-20"
-              tabIndex="0"
-              data-testid="status-dropdown"
-            >
-              <Badge state={versionStatus} />
-              <Icon
-                className="m-12"
-                width={20}
-                height={20}
-                kind="chevron-down"
-              />
-            </button>
-            <div className="Dropdown--container">
-              <ul className="Dropdown--list">
-                {versionStatusList.map(versionStatusValue => (
-                  <button
-                    onClick={() => onVersionStatusChange(versionStatusValue)}
-                    type="button"
-                    className="Dropdown--item"
-                    tabIndex="0"
-                    key={versionStatusValue}
-                  >
-                    <Badge state={versionStatusValue} />
-                  </button>
-                ))}
-              </ul>
-            </div>
-          </div>
-        </div>
-        <label
-          className="Form--Label row-3 cell-3 text-right Form--Label-required"
-          htmlFor="file_type"
-        >
-          Document Type:
-        </label>
-        <fieldset name="file_type" className="row-3 cell-9">
+      </Form.Field>
+      <Form.Field>
+        <label>Approval Status:</label>
+        <Dropdown
+          selection
+          fluid
+          options={options}
+          value={versionStatus}
+          placeholder="Choose an option"
+          onChange={(e, {value}) => {
+            onVersionStatusChange(value);
+          }}
+        />
+      </Form.Field>
+      <Form.Field required>
+        <label htmlFor="file_type">Document Type:</label>
+        <fieldset name="file_type">
           {Object.keys(fileTypeDetail).map(item => (
             <SelectElement
```

Comment:
make each SelectElement in you fileTypeDetail iteration a Form.Field

Let's also update the `select` prop to just use the value and not the event since we called it with the raw value in the SelectElement component
```
  < Form.Field>
      <SelectElement select={val => onFileTypeChange(val}  />
   < /Form.Field>
```